src/app/navbar/navbar.component.scss

```diff
@@ -1,34 +1,17 @@
-@import "../styles/base/_variables";
-@import "../styles/components/_buttons";
+@import "../styles/main";
 
-.profile-pic{
+.profile-pic {
     height: $profile-pic-height;
```

Comment:
i would use `max-height` & `max-width` instead for images, to prevent image distortion